src/mdtf_info.py

```diff
@@ -50,8 +62,8 @@ def _load_one_json(pod):
             d = _load_one_json(p)
             assert d
         except Exception as exc:
-            bad_pods.append(p)
-            continue
+            _log.critical(exc) # exc_info=sys.exc_info())
+            exit(1)
```

Comment:
I think it would be better to scan all of the PODs at once and test if `len(bad_pods) > 0`.  That way the end user doesn't have to iterate over and over in the event there are multiple PODs that failed to import.

Reply:
Good point; spun this out into its own PR in https://github.com/NOAA-GFDL/MDTF-diagnostics/pull/183.